.github/workflows/node-zxc-build-release-artifact.yaml

```
@@ -374,57 +437,156 @@ jobs:
+              sudo apt update
+            echo "::endgroup::"
+            echo "::group::Installing GnuPG Tools"
+              sudo apt install -y gnupg2
+            echo "::endgroup::"
+          fi
+
+      - name: Import GPG key
+        id: gpg_key
+        uses: crazy-max/ghaction-import-gpg@d6f3f49f3345e29369fe57596a3ca8f94c4d2ca7 # v5.4.0
+        if: ${{ inputs.dry-run-enabled != true && !cancelled() && !failure() }}
+        with:
+          gpg_private_key: ${{ secrets.svcs-gpg-key-contents }}
+          passphrase: ${{ secrets.svcs-gpg-key-passphrase }}
+          git_config_global: true
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_tag_gpgsign: true
+
+      - name: Setup Java
+        uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v3.12.0
+        with:
+          distribution: ${{ inputs.java-distribution }}
+          java-version: ${{ inputs.java-version }}
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@ef76a971e2fa3f867b617efd72f2fbd72cf6f8bc # v2.8.0
+        with:
+          gradle-version: ${{ inputs.gradle-version }}
+          gradle-home-cache-includes: |
+            caches
+            notifications
+            jdks
+
+      - name: Restore Build Version
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
+        with:
+          fail-on-cache-miss: true
+          path: version.txt
+          key: node-build-version-${{ needs.validate.outputs.version }}-${{ github.sha }}
+
+      - name: Gradle Update Version (Snapshot)
+        uses: gradle/gradle-build-action@ef76a971e2fa3f867b617efd72f2fbd72cf6f8bc # v2.8.0
+        if: ${{ inputs.version-policy != 'specified' && !cancelled() && !failure() }}
+        with:
+          gradle-version: ${{ inputs.gradle-version }}
+          arguments: versionAsSnapshot --scan
+
+      - name: Gradle Assemble
+        id: gradle-build
+        uses: gradle/gradle-build-action@ef76a971e2fa3f867b617efd72f2fbd72cf6f8bc # v2.8.0
+        with:
+          gradle-version: ${{ inputs.gradle-version }}
+          arguments: assemble --scan
+
+      - name: Gradle Version Summary
+        uses: gradle/gradle-build-action@ef76a971e2fa3f867b617efd72f2fbd72cf6f8bc # v2.8.0
+        with:
+          gradle-version: ${{ inputs.gradle-version }}
+          arguments: githubVersionSummary --scan
+
+      - name: Gradle Maven Central Release
+        uses: gradle/gradle-build-action@ef76a971e2fa3f867b617efd72f2fbd72cf6f8bc # v2.8.0
+        if: ${{ inputs.dry-run-enabled != true && inputs.version-policy == 'specified' && !cancelled() && !failure() }}
+        env:
+          OSSRH_USERNAME: ${{ secrets.svcs-ossrh-username }}
+          OSSRH_PASSWORD: ${{ secrets.svcs-ossrh-password }}
+        with:
+          gradle-version: ${{ inputs.gradle-version }}
+          arguments: -p hedera-node/hedera-evm releaseMavenCentral --scan -PpublishSigningEnabled=true
```

To be consistent in the pipelines, I would run all the tasks through the "root" umbrella build. I.e., use the `:` separated task path to address the target task instead of using `-p` to point at a directory:

```suggestion
          arguments: ":hedera-node:hedera-evm:releaseMavenCentral --scan -PpublishSigningEnabled=true"
```

(I know I used -p myself before but that was because I wanted to change as little as possible in the pipelines in the first step)